test/IRGen/unmanaged_objc_throw_func.swift

```diff
@@ -4,12 +4,14 @@
 
 import Foundation
 
-@objc protocol SR_9035_P {
+// https://github.com/apple/swift/issues/51538
+
+@objc protocol P {
   func returnUnmanagedCFArray() throws -> Unmanaged<CFArray>
 }
 
-// CHECK-LABEL: define hidden swiftcc %TSo10CFArrayRefa* @"$s25unmanaged_objc_throw_func9SR_9035_CC22returnUnmanagedCFArrays0G0VySo0H3RefaGyKF"
-@objc class SR_9035_C: NSObject, SR_9035_P {
+// CHECK-LABEL: define hidden swiftcc %TSo10CFArrayRefa* @"$s25unmanaged_objc_throw_func1CC22returnUnmanagedCFArrays0F0VySo0G3RefaGyKF"
+@objc class C: NSObject, P {
   func returnUnmanagedCFArray() throws -> Unmanaged<CFArray> {
```

Comment:
The slight divergence in the `Unmanaged` and `CFArray` result portions of the mangled signature, `s0G0V → s0F0V` and `So0H3Refa → So0G3Refa` respectively, which exercise [word substitutions](https://github.com/apple/swift/blob/main/docs/ABI/Mangling.rst#identifiers), are expected and due to `C` having one less word—none, in fact—than the old `SR_9035_C`.